extensions/wikia/AssetsManager/config.php

```diff
@@ -2539,13 +2539,6 @@
 	]
 ];
 
-$config['special_user_activity_css'] = [
```

Comment:
We should remove the call from the controller to load this as well: https://github.com/Wikia/app/blob/dev/extensions/wikia/UserActivity/SpecialUserActivityController.class.php#L44-L49